addons/mail/static/src/components/composer/composer.xml

```diff
@@ -48,12 +48,24 @@
                         </t>
                     </div>
                 </t>
+                <t t-if="composer.thread and composer.thread.mass_mailing">
+                    <div class="o_Composer_subject">
+                        <!-- // FIXME this whould not be needed if input bindings is handled in a new component (task-2284290) -->
```

Comment:
remove comment from here, it's not clear it's related to the on-input only and the input tag itself will always be needed